### ISSUE TITLE
I have a question about starting the game

### ISSUE THREAD

**Issue by DarkDakurai** [mcclure/emu-coop#29]
Me and my cousin wanted to play super metroid, do we have to start on the same file? Like, do we both have to pick file A for example?

**Comment by mcclure** [comment]
Nope, it shouldn't matter. In fact you can be using slightly different ROMs as long as they're based on the same version of the game. All that matters is that the RAM offsets are the same in both files, which it will be once the game starts.

By the way, be aware of http://multitroid.com/ , it is receiving more active development.
